litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/TableData.tsx

```diff
@@ -71,6 +73,23 @@ const TableData: React.FC<TableDataProps> = ({ data, deleteRow }) => {
     setIsModalOpen(false);
   };
 
+  // States for PopOver to display schedule details
+  const [
+    popAnchorElSchedule,
+    setPopAnchorElSchedule,
+  ] = React.useState<null | HTMLElement>(null);
+  const isOpenSchedule = Boolean(popAnchorElSchedule);
+  const idSchedule = isOpenSchedule ? 'simple-popover' : undefined;
```

Comment:
can a constant be used for this string id `simple-popover`

Reply:
It won't be changing, it is only for the popover component.